sensor/Dockerfile.raspberrypi4-64

```diff
@@ -0,0 +1,25 @@
+FROM balenalib/raspberrypi3-python:3-build
```

Comment:
@chrisys I would make this `Dockerfile.aarch64`, because then the `raspberrypi3-64` can use it as well (that's a device type we have too, just too rarely used:)